build/steps-build-windows.yml

```diff
@@ -0,0 +1,50 @@
+﻿parameters:
+- name: pathToSrc
+  type: string
+  default: ''
+- name: solutionFileName
+  type: string
+  default: ''
+- name: solutionName
+  type: string
+  default: ''
+
+steps:
+- template: templates/dotnet-install.yml
+
+#-if false
+- download: current
+  condition: eq('GeneratedApp.sln', '${{ parameters.solutionFileName }}')
+  artifact: GeneratedApp
+#-endif
+
+- script: >
+    cd ${{ parameters.pathToSrc }}/app/${{ parameters.solutionName }}.Windows
+
```

Comment:
why the empty line here?

Reply:
When we use `>` it converts line break to spaces, so to have an actual line break we need this.